testkit/examples/timestamping/proto/mod.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Module of the rust-protobuf generated files.
+
+// For protobuf generated files.
+#![allow(bare_trait_objects)]
+#![allow(renamed_and_removed_lints)]
```

Comment:
Is it possible to teach rust-protobuf print this on its own? I'm not a fan of boilerplate that we _have_ to keep forever in out files because someone else does something stupid. rust-protobuf seems to output [quite a few](https://github.com/stepancheg/rust-protobuf/blob/43641931ad427c0d2874f298d6d86c9c128e47a2/protobuf/src/descriptor.rs#L4-L18) clippy annotations, why don't we ask them nicely to add some more?

Reply:
Unfortunately no, it is known issue. Maybe it is even fixed, but not released yet.

Reply:
Why not just use current workaround without patching generated files? It seems simpler.

Unfortunately we cannot add this annotations to file which we `import!` for the same reason this workaround with `import!`ing piece of the mod file exists.

Reply:
I don't think we should waste our limited resources, so current solution is preferable.

Reply:
[This](https://github.com/stepancheg/rust-protobuf/issues/331) [(PR)](https://github.com/stepancheg/rust-protobuf/pull/332) seems to be the tracking issue for this in rust-protobuf. It seems to be blocked by clippy's interface insanity w.r.t. stable vs. nighty and scoped vs. legacy attributes.